applications/smart-distancing/docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
+

```

Comment:
It's totally up to you. Just to clarify in case there is a misunderstaninding, I am not at all offended. It's very difficult to actually offend me so I'm saying: don't hold back if you have concerns :)  Likewise, if I offend you, let me know. I am better with machines than people and it's common that I step on toes without realizing it.

Reply:
It's same here so we are good, I guess you are an INTP/J :D

Reply:
Haha, yes. I don't remember which, but it's one of those. It's been a while since I took that test.